templates/dialogs/ffg-sheet-options.html

```diff
@@ -1,15 +1,16 @@
 <div class="dialog-content sheet-options-dialog">
+    {{{ json content }}}
```

Comment:
Everything looks good to me. Probably just cleanup the logging.

Reply:
🤦 fixed